frontend/src/routes/Governance/policies/policy-details/PolicyDetailsResults.tsx

```diff
@@ -228,15 +241,10 @@ export default function PolicyDetailsResults() {
                 <span className="link-disabled">{`- ${t('View details')}`}</span>
               </Tooltip>
             )
-            const templateExists = !(
-              prunedMessage.includes('Failed to create policy template') ||
-              prunedMessage.includes('check if you have CRD deployed') ||
-              prunedMessage.includes('Dependencies were not satisfied')
-            )
             return (
               <div>
                 {/* message may need to be limited to 300 chars? */}
-                {prunedMessage} {templateExists && templateLink}
+                {prunedMessage} {templateExists(prunedMessage) && templateLink}
```

Comment:
I thought we disabled the link

Reply:
Apparently only if the user doesn't have permission.